react/features/analytics/actions.js

```diff
@@ -0,0 +1,119 @@
+import JitsiMeetJS from '../base/lib-jitsi-meet';
+import { loadScript } from '../base/util';
+
+const logger = require('jitsi-meet-logger').getLogger(__filename);
+
+/**
+ * Loads the analytics scripts and inits JitsiMeetJS.analytics by setting
+ * permanent properties and setting the handlers from the loaded scripts.
+ * NOTE: Has to be used after JitsiMeetJS.init. Otherwise analytics will be
+ * null.
+ *
+ * @param {Store} store - The redux store in which the specified {@code action}
+ * is being dispatched.
+ * @returns {void}
+ */
+export function initAnalytics({ getState }) {
+    window.analyticsHandlers = [];
+
+    const { analytics } = JitsiMeetJS;
+    const { enabled, scriptURLs } = getState()['features/analytics'];
+    const config = getState()['features/base/config'];
+
+    if (!enabled || !analytics) {
+        return;
+    }
+
+    const machineId = JitsiMeetJS.getMachineId();
+    const handlerConstructorOptions = {
+        product: 'lib-jitsi-meet',
```

Comment:
don't we want to customize this?

Reply:
I suppose we need to update handlerConstructorOptions if we are moving between deployments, which is possible with mobile. This reply to the question about the loading it once and reusing it.